README.Rmd

```diff
@@ -133,14 +133,15 @@ On your own data you can easily achieve this with `dplyr`'s `filter()` function.
 
 Due to the fact that variances increase with increasing raw intensities, statistical tests would have a bias towards lower-intensity peptides or proteins. Therefore you should log2 transform your data to correct for this mean-variance relationship. We do not need to do this for the synthetic data as it is already log2 transformed. For your own data just use `dplyr`'s `mutate()` together with `log2()`. 
 
-In addition to filtering and log2 transformation it is also advised to median normalise your data to equal out small differences in overall sample intensities that result from unequal sample concentrations. **protti** provides the `median_normalisation()` function for this purpose. This function generates an additional column called `normalised_intensity_log2` that contains the normalised intensities.
+In addition to filtering and log2 transformation it is also advised to normalise your data to equal out small differences in overall sample intensities that result from unequal sample concentrations. **protti** provides the `normalise()` function for this purpose. We generally recommend using median normalisation (`method = "median"`). This function generates an additional column called `normalised_intensity_log2` that contains the normalised intensities.
```

Comment:
Not sure if I'd say that we recommend to use median normalisation. For some applications there's probably better options. I'd maybe say something along the lines of "For this example we will use median normalisation".